Instructions/Labs/13-implement-transitions-in-tsql.md

```diff
@@ -241,6 +241,62 @@ END CATCH
 
 8. Switch to the query window that contains code to select the most recently modified customer and run it to verify that a record for *Ann Othercustomer* has been inserted. The transaction succeeded and was not rolled back, even though an error subsequently occurred.
 
+## Explore transaction concurrency
+
+On-premises SQL Server has a default isolation level of **READ_COMMITTED_SNAPSHOT_OFF**. This level of isolation will hold locks on rows while a transaction is acting on it. For example, inserting a customer into the customer table. If the update takes a long time to run, any queries on that table will be blocked from running until the transaction is committed or rolled back.
+
+1. Start a new instance of Azure Data Studio by right-clicking in the task bar on Azure Data Studio  and then select **Azure Data Studio**.
+2. There should be two instances running, rearrange the Azure Data Studio windows so that both windows can be seen on the same screen.
+
+![A screenshot showing two instances of Azure Data Studio side by side.](./images/side-by-side.png)
+
+3. Highlight the `BEGIN TRANSACTION` and first `INSERT` statements in the first window.
+4. In the second window enter this query.
+
+```
+SELECT COUNT(*) FROM SalesLT.Customer
+```
+
+5. In the query window select **&#x23f5;Run** at the top of the query window, or press the <kbd>F5</kbd> key to run the query.
+
+6. Note the number of counted rows. Also note how long the query took to complete. 
+7. In the first window with the selected statements select **&#x23f5;Run**. The message is:
+
+```
+(1 row affected)
+Total execution time: 00:00:01.921
+```
+
+As this is in a transaction that hasn't been committed, running the SQL query in the other window will be blocked.
+
+8. In the other query window select **&#x23f5;Run**.

```

Comment:
They already know how to run code - Tell them which button the first time, and then for subsequent scripts, just tell them to run it.